src/main/java/dtb/lego/truck/rest/data/events/acquisition/boundary/EventHistoryResource.java

```diff
@@ -0,0 +1,83 @@
+package dtb.lego.truck.rest.data.events.acquisition.boundary;
+
+import dtb.lego.truck.rest.component.events.entity.events.Event;
+import dtb.lego.truck.rest.data.events.acquisition.control.SensorDataHandler;
+import dtb.lego.truck.rest.errors.InputValidator;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.ZonedDateTime;
+import java.util.Collection;
+
+@RestController
+@Api(description = "Operations pertaining the acquisition of historical data of the events")
+public class EventHistoryResource {
+
+    @Autowired
```

Comment:
hint: @Autowired works fine but c'tor injection is preferred and has some advantages.